pkgvars

```diff
@@ -1,6 +1,6 @@
 pkgname=citus
 pkgdesc='Citus (Open-Source)'
-pkglatest=9.4.5.citus-1
-releasepg=11,12
-nightlypg=11,12
+pkglatest=9.5.6.citus-1
+releasepg=11,12,13
+nightlypg=12,13
```

Comment:
We already disabled the nightly builds. Does it really make sense to update the supported PG versions here?